src/SDKs/Monitor/Management.Monitor/Generated/Management/Monitor/DiagnosticSettingsOperations.cs

```diff
@@ -0,0 +1,803 @@
+                {
+                    _httpResponse.Dispose();
+                }
+                throw ex;
+            }
+            // Create Result
+            var _result = new AzureOperationResponse();
+            _result.Request = _httpRequest;
+            _result.Response = _httpResponse;
+            if (_httpResponse.Headers.Contains("x-ms-request-id"))
+            {
+                _result.RequestId = _httpResponse.Headers.GetValues("x-ms-request-id").FirstOrDefault();
+            }
+            if (_shouldTrace)
+            {
+                ServiceClientTracing.Exit(_invocationId, _result);
+            }
+            return _result;
+        }
+
+        /// <summary>
+        /// Updates an existing DiagnosticSettingsResource tags. To update other fields
+        /// use the CreateOrUpdate method.
+        /// </summary>
+        /// <param name='resourceUri'>
+        /// The identifier of the resource.
+        /// </param>
+        /// <param name='diagnosticSettingResource'>
+        /// Parameters supplied to the operation.
+        /// </param>
+        /// <param name='name'>
+        /// The name of the diagnostic setting.
+        /// </param>
+        /// <param name='customHeaders'>
+        /// Headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        /// <exception cref="ErrorResponseException">
+        /// Thrown when the operation returned an invalid status code
+        /// </exception>
+        /// <exception cref="SerializationException">
+        /// Thrown when unable to deserialize the response
+        /// </exception>
+        /// <exception cref="ValidationException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <return>
+        /// A response object containing the response body and response headers.
+        /// </return>
+        public async Task<AzureOperationResponse<DiagnosticSettingsResource>> UpdateWithHttpMessagesAsync(string resourceUri, DiagnosticSettingsResource diagnosticSettingResource, string name, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (resourceUri == null)
+            {
+                throw new ValidationException(ValidationRules.CannotBeNull, "resourceUri");
+            }
+            if (diagnosticSettingResource == null)
+            {
+                throw new ValidationException(ValidationRules.CannotBeNull, "diagnosticSettingResource");
+            }
+            if (name == null)
+            {
+                throw new ValidationException(ValidationRules.CannotBeNull, "name");
+            }
+            string apiVersion = "2016-09-01";
```

Comment:
Vinicius is changing the Swagger spec to remove the new diagnostic settings API.
I got his changes locally and generated the code, I am updating this PR for the sake of time.